assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -723,7 +723,7 @@ che.cors.allow_credentials=false
 # which does not contain any {prod-short}-specific workspace descriptor
 # Multiple plugins must be comma-separated, for example:
 # `pluginFooPublisher/pluginFooName/pluginFooVersion,pluginBarPublisher/pluginBarName/pluginBarVersion`
-che.factory.default_plugins=NULL
+che.factory.default_plugins=redhat/vscode-commons/latest
```

Comment:
> for devWorkspaces, the resolver will handle dependencies and then will be able to load only one instance of vscode-commons

I'm not sure I got how it's going to work. Devfile does not seem to be available for it https://che-plugin-registry-main.surge.sh/v3/plugins/redhat/vscode-commons/latest/devfile.yaml

Reply:
https://che-plugin-registry-main.surge.sh/v3/plugins/redhat/vscode-commons/latest/meta.yaml
https://che-plugin-registry-main.surge.sh/v3/plugins/redhat/vscode-commons/latest/che-theia-plugin.yaml

Reply:
Well, Dashboard does not know plugin type before it loads it. And as far as I understand, Dashboard will try to load the corresponding devfile.

I think it should be OK to temporary disable default plugins for devworkspaces since we don't have any ATM, the last we used was machine exec.
Would it work for you?

Reply:
> Well, Dashboard does not know plugin type before it loads it

sure, but https://che-plugin-registry-main.surge.sh/v3/plugins/index.json will tell that it is a `VS Code extension`

```json
{
"id": "redhat/vscode-commons/latest",
"description": "Base utilities for Red Hat extensions. Notably controls telemetry settings.",
"displayName": "Red Hat Commons",
"links": {
"self": "/v3/plugins/redhat/vscode-commons/latest",
"plugin": "/v3/plugins/redhat/vscode-commons/latest/che-theia-plugin.yaml"
},
"name": "vscode-commons",
"publisher": "redhat",
"type": "VS Code extension",
"version": "latest"
},
```

> I think it should be OK to temporary disable default plugins for devworkspaces since we don't have any ATM, the last >we used was machine exec.
> Would it work for you?

it might work but if dashboard is reading index.json of the plug-in registry then it shouldn't do anything for VS Code extension in case of DevWorkspaces

Reply:
che-server workspace: we need that plug-in
devWorkspace: we don't need that plug-in

Reply:
Do not request plugins for devworkspaces https://github.com/eclipse-che/che-dashboard/pull/230

Reply:
> it might work but if dashboard is reading index.json of the plug-in registry then it shouldn't do anything for VS Code extension in case of DevWorkspaces


dashboard knows nothing about index.json since it's able to request plugin configuration directly. it's +1 redundant request we can and do avoid